### PR TITLE
Enable disabled test [SCHOOL-99]

### PR DESCRIPTION
**Pull request by Camus1859**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
+root = true
```
Comment (nbey) [inline .editorconfig]: Let's move this file into it's own PR (I think you have one created already)

**Review by themightychris — APPROVED** [review]

**Review by nbey — APPROVED** [review]

**Comment by themightychris** [comment]
This PR seems good to merge aside from whatever 500 crash is happening

**Review by themightychris — APPROVED** [review]

File: cypress/integration/login_admin.js [hunk 2]
```diff
@@ -10,15 +10,15 @@ describe.skip('Admin login test', () => {
 
         // should be visible after Log In click
         cy.get('#login-modal').should('not.be.visible');
-        cy.contains('Log In').click();
+        cy.get('.slate-omnibar.mobile-hidden').contains('Log In').click();
```
Comment (themightychris) [inline cypress/integration/login_admin.js:13]: @Camus1859 in the future, it's better to augment the source application with semantic `data-*` attributes for tests to anchor to than to rely on any presentation-centric CSS tags
